### PR TITLE
chore: Use latest image tag

### PR DESCRIPTION
**Pull request by spolti**

### CODE REVIEW

**Review by rafvasq — APPROVED** [review]
/lgtm

File: Dockerfile [hunk 1]
```diff
@@ -134,7 +134,7 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
 ###############################################################################
 # Stage 3: Copy binaries only to create the smallest final runtime image
 ###############################################################################
-FROM registry.access.redhat.com/ubi8/ubi-micro:8.8 as runtime
+FROM registry.access.redhat.com/ubi8/ubi-micro:latest as runtime
```
Comment (ckadner) [inline Dockerfile:137]: this is not good for reproducibility

**Review by ckadner — COMMENTED** [review]
We should always use a specific version to ensure reproducible builds

File: Dockerfile (hunk 1 above)
Comment (ckadner) [reply Dockerfile:137]: I created PR #43 to move up to UBI9 :-)

**Pull request merged**